Runtime/Model/Attributes/MachineAttributeProvider.cs

```diff
@@ -82,7 +82,11 @@ private void IncludeOsInformation(IDictionary<string, string> attributes)
         }
         private void IncludeGraphicCardInformation(IDictionary<string, string> attributes)
         {
-
+            // if a graphic card is not available
+            if (!SystemInfo.graphicsDeviceType)

```

Comment:
here are all possible types: https://docs.unity3d.com/ScriptReference/Rendering.GraphicsDeviceType.html 

there is no option it can be true. 


I will make sure situation you're talking about won't happen - I will check if the graphicsDeviceType is null